spec/jobs/searchgov_urls_job_spec.rb

```diff
@@ -0,0 +1,49 @@
+# require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe SearchgovUrlsJob, type: :job do
```

Comment:
All this test are failing, please fix. From here https://app.circleci.com/pipelines/github/GSA/search-gov/2464/workflows/5f45e19f-9286-4aeb-8842-15767f8df4c1/jobs/20669

```
Finished in 59.03 seconds (files took 6.99 seconds to load)
1005 examples, 7 failures, 1 pending

Failed examples:

rspec ./spec/jobs/searchgov_urls_job_spec.rb:43 # SearchgovUrlsJob#perform handles an exception
rspec ./spec/jobs/searchgov_urls_job_spec.rb:24 # SearchgovUrlsJob#perform calls upload_and_index on the BulkUrlUploader
rspec ./spec/jobs/searchgov_urls_job_spec.rb:10 # SearchgovUrlsJob#perform sets the time started
rspec ./spec/jobs/searchgov_urls_job_spec.rb:31 # SearchgovUrlsJob#perform logs the results
rspec ./spec/jobs/searchgov_urls_job_spec.rb:36 # SearchgovUrlsJob#perform logs the correct information
rspec ./spec/jobs/searchgov_urls_job_spec.rb:15 # SearchgovUrlsJob#perform sets the total count
rspec ./spec/jobs/searchgov_urls_job_spec.rb:19 # SearchgovUrlsJob#perform creates a new BulkUrlUploader
```